rsconnect/main.py

```diff
@@ -438,8 +438,7 @@ def write_manifest():
               help='Path to python interpreter whose environment should be used. ' +
                    'The python environment must have the rsconnect package installed.')
 @click.option('--conda', '-C', is_flag=True,
-              help='Force freezing the current environment using pip instead of conda, when conda is not supported on '
-                   'RStudio Connect (version<=1.8.0).')
+              help='Use conda to deploy (unsupported for versions <=1.8.0)')
```

Comment:
nit: I initially read this as requiring Connect 1.8.0. Maybe just my mistake, but it might be clearer as `requires Connect 1.8.2 or later`.

Reply:
Note that I'm hoping this flag/message never makes it to release. Next time we cut a release I'd like conda supported by default. I'd also like to implement automatic detection of conda support via the server_settings endpoint.

Reply:
Updated, thanks!